microversetrial-3.html

```diff
@@ -0,0 +1,24 @@
+<nav id="navbar">
+  <ul>
+    <li><a href="#welcome-section">welcome section</a></li>
+    <li></li>
+    <li><a href="#project section">project section</a></li>
+    <li></li>
+  </ul>
```

Comment:
it would look much better if you do not leave a vacant list item between the filled ones

Reply:
kindly remember to start with `kindly` or `please` so that the student does not think you are giving him an order.